src/Evolution/DiscontinuousGalerkin/Actions/ComputeTimeDerivative.hpp

```diff
@@ -467,7 +478,8 @@ ComputeTimeDerivative<Dim, EvolutionSystem, DgStepChoosers, LocalTimeStepping>::
           number_of_grid_points +
       // Different number of grid points. See explanation above where
       // num_face_temporary_grid_points is defined
-      VarsFaceTemporaries::number_of_independent_components *
+      (VarsFaceTemporaries::number_of_independent_components +
+       VarsMortarTemporaries::number_of_independent_components) *
```

Comment:
Since the mortar can have more points than the interface, do you have to worry that `num_face_temporary_grid_points` is too small? I think if you were to do `number_of_grid_points` then the buffer might be larger than necessary, but would never be too small. That is, reading the comment about face points, what happens if your neighbor has more face points than you? Wouldn't in that case this allocation be too small?

Edit: aha! this `Mortar` is the tags we will be projecting _to_ the mortar. Could you name this name this `DgPackagedDataVarsOnFace` instead of `VarsMortarTemporaries`?

Reply:
Also renamed the itself variable to better reflect what it was